common/app/views/fragments/affiliateLinksDisclaimer.scala.html

```diff
@@ -1,24 +1,27 @@
-@(contentType: String)
```

Comment:
A thought for the future, but I wonder if this Twirl template should be migrated to DCR, and Frontend just passes along a flag `showAffiliateLinksDisclaimer: boolean`.

Reply:
Oh yes I'd forgotten about galleries, although I suppose they can always use the legacy twirl template until they're migrated over, whilst DCR can start using it's own disclaimer component.